Datadog.Trace.proj

```diff
@@ -2,12 +2,12 @@
   <PropertyGroup>
     <BuildInParallel Condition="'$(BuildInParallel)' == ''">true</BuildInParallel>
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
-    <TracerHomeDirectory Condition="'$(TracerHomeDirectory)' == ''">$(MSBuildThisFileDirectory)src\bin\tracer-home</TracerHomeDirectory>
+    <TracerHomeDirectory Condition="'$(TracerHomeDirectory)' == ''">$(MSBuildThisFileDirectory)src\bin\windows-tracer-home</TracerHomeDirectory>
```

Comment:
@dszmigielski keep an eye if these will affect the PR to generate the build artifacts.

Reply:
It should not, but I will test it after this one is merged.

Reply:
The `packages` build should be fine because it sets the `TracerHomeDirectory` property itself, but I had to issue a bug fix for local builds if you don't set it: https://github.com/DataDog/dd-trace-dotnet/commit/1acaff04b3d425e4da13bb3745381cf275cbc612